tests/test_web_sendfile_functional.py

```diff
@@ -238,47 +243,60 @@ async def handler(request):
     app.router.add_get("/", handler)
     client = await aiohttp_client(app)
 
-    resp = await client.get("/")
+    resp = await client.get("/", headers={"Accept-Encoding": accept_encoding})
```

Comment:
We should probably keep this test as is and add a new one that sends the header so we cover both cases or params to do both here

Reply:
Would adding `headers=None` or just `accept_encoding=None` as another test do the trick?

Reply:
`headers=None` should preserve the existing test behvior

Reply:
Done